app/scripts/abotypen/overview/abotypenoverview.controller.js

```diff
@@ -53,10 +53,10 @@ angular.module('openolitor-admin')
           return value;
         });
 
-        $scope.emailAddresses = _($scope.filteredEntries)
+        $scope.abotypIdsMailing = _($scope.filteredEntries)
```

Comment:
Where do you use this newly created property?

Reply:
this was a bug I found while testing the emailing. This property was wrong and the ids were not transfer to the directive. The property needed to match the one in the html

![image](https://user-images.githubusercontent.com/13901369/155494415-f5213cf3-2e0c-4e5e-af17-4a4fad65621c.png)